form/form.go

```diff
@@ -147,7 +170,7 @@ type IOPrompter struct {
 	Out io.Writer
 }
 
-// Prompt implements Prompter. The prompt is generated from attr based on
+// Prompt implements Prompter.Prompt. The prompt is generated from attr based on
```

Comment:
// Prompt implements Prompter.Prompt by writing the field information
// to p.Out, then reading input from p.In. If p.In is a terminal and the
// attribute is secret, echo will be disabled.
?